tests/foreman/api/test_notifications.py

```diff
@@ -0,0 +1,230 @@
+"""Test for Notifications
```

Comment:
```suggestion
"""Test class for Notifications API
```

Reply:
Fixed in updated commit https://github.com/SatelliteQE/robottelo/commit/2c5a24d600f8941509ef10833a9631e3cd1d6a3e